### ISSUE TITLE
Remove dependency on tensorflow.contrib

### ISSUE THREAD

**Issue by ian-r-rose** [CityOfLosAngeles/count-machine#19]
The most recent versions of tensorflow have [removed](https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md) `tensorflow.contrib`. Can we remove this dependency?

**Comment by ian-r-rose** [comment]
Any thoughts about this @vahedi25411?

**Comment by vahedi25411** [comment]
My code is not dependent on this, but it has been used in some files and I should spend some time to review them and figure out how can I change it.

**Comment by ian-r-rose** [comment]
Makes sense. How much of the current codebase is not used? Can we remove a large chunk of it (including the stuff depending on `tensorflow.contrib`)?

**Comment by vahedi25411** [comment]
They are mostly the parts that are used for training and I am not sure that we can remove them especially if we want to train our model for the scooter.

**Comment by ian-r-rose** [comment]
Is the training code currently up-to-date and tested? Or would there be work to update that with the most recent version of the count components?

Tensorflow seems to have a script which attempts to [auto-upgrade](https://www.tensorflow.org/guide/upgrade) to 2.0, though I don't really know how well it works.

**Comment by vahedi25411** [comment]
Not really. I am not even sure whether we are going to use it or not. So I prefer to work on it when we are 100 percent sure that we need it.

It would be a good idea to update the code using the script and test it, but I think we should create another environment because the prerequisites also change for this version.

**Comment by hunterowens** [comment]
status on this?

**Comment by ian-r-rose** [comment]
Still needs to be done.

**Comment by ian-r-rose** [comment]
@vahedi25411 do you have a sense of how much work this would be?
